containers/R_Python/r.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import print_function
+from sklearn.externals import joblib
+import rpc
+import os
+import numpy as np
+import scipy as sp
+import warnings
+warnings.filterwarnings("ignore", category=FutureWarning)
+from pandas import *
+import pandas.rpy.common as com
```

Comment:
This module has been moved to the `rpy2` package. See the earlier comment about this.